frontstage/views/register.py

```diff
@@ -281,10 +274,10 @@ def register_enter_your_details():
 
         # party_service_url = app.config['API_GATEWAY_PARTY_URL'] + 'respondents'
         party_service_url = app.config['RAS_PARTY_POST_RESPONDENTS'].format(app.config['RAS_PARTY_SERVICE'])
-        logger.debug("Party service URL is: {}".format(party_service_url))
+        logger.debug('"event" : "get party url", "Party service URL is" : "{}"'.format(party_service_url))
 
         result = requests.post(party_service_url, headers=headers, data=json.dumps(registration_data))
-        logger.debug("Response from party service is: {}".format(result.content))
+        logger.debug('"event" : "party url", "Response from party service is" : "{}"'.format(result.content))
```

Comment:
same as above here. I only commented the top stuff before so you might need to go through these files too to make it consistent